src/backend/opengl/CMakeLists.txt

```diff
@@ -208,6 +208,10 @@ ELSE(USE_SYSTEM_GLBINDING)
     ADD_DEPENDENCIES(forge glbinding)
 ENDIF(USE_SYSTEM_GLBINDING)
 
+SET_TARGET_PROPERTIES(forge PROPERTIES
+                      VERSION ${FG_VERSION}
+                      SOVERSION 0)
```

Comment:
We are using `AF_VERSION_MAJOR` for `SOVERSION` in arrayfire. Can you use `FG_VERSION_MAJOR` here for consistency ?